libvast/vast/taxonomies.hpp

```diff
@@ -0,0 +1,50 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#pragma once
+
+#include "vast/fwd.hpp"
+
+#include <caf/meta/type_name.hpp>
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace vast {
+
+using concepts_t = std::unordered_map<std::string, std::vector<std::string>>;
+using models_t = std::unordered_map<std::string, std::vector<std::string>>;
```

Comment:
```suggestion
using concepts_type = std::unordered_map<std::string, std::vector<std::string>>;
using models_type = std::unordered_map<std::string, std::vector<std::string>>;
```

The `_t` strikes me as a C-ism. In C++ we use `_type` if we can, and only resort to `_t` if need be.